src/devices/DCMotor/DCMotor.cs

```diff
@@ -11,21 +11,39 @@ namespace Iot.Device.DCMotor
 {
     public abstract class DCMotor : IDisposable
     {
-        protected GpioController _controller;
+        /// <summary>
+        /// Controller related with operations on pins
+        /// </summary>
+        protected GpioController Controller;
```

Comment:
Why is this protected? Do you use it anywhere? If not, this should probably be private and readonly

Reply:
Yes, this is used in two of the implementations.